release/models/wifi/.spec.yml

```diff
@@ -0,0 +1,7 @@
+- name: openconfig-wifi-mac
```

Comment:
do we need a spec file here if we have in the `mac` subdir?

Reply:
Don't think so. Probably left here inadvertently. Will remove.